### DISCUSSION TITLE
Product title not coming on Swissup ajaxpro quickview popup

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#56] [1. Help]
Hi @vovayatsyuk 

The product title is not coming on the quick view popup, we are using the Swissup_Ajaxpro module with the breeze theme.
Can you please look into this?
https://mcprod.mishimoto.com/automotive/hose-clamps.html

Thanks

**Comment by vovayatsyuk** [comment]
I believe it was fixed in ajaxpro 1.6.10 Which version do you use?

**Reply by pk-pankaj** [reply]
We are using ajaxpro 1.6.8.
